client/src/components/views/PlantsPage/PlantPageOfferts.tsx

```diff
@@ -42,90 +45,40 @@ const ImageContainer = styled.div`
   height: 86%;
 `;
 
-//Temporary Array//
 export const PlantPageOfferts = () => {
-  const cards = [
-    {
-      id: "EFFF3Eessdf",
-      name: "Chwast",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
+  const { response } = useFetchData({
+    url: `/offers`,
+    method: "GET",
+    headers: {
+      accept: "*/*",
+      "Content-Type": "application/json",
     },
-    {
-      id: "EFFgdssdEessdf",
-      name: "Mandragora",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "EFFgdss",
-      name: "ARG",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "EdsfFFgdss",
-      name: "GFZfg",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "ESGEgg",
-      name: "Chwast",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "ESGEg5g",
-      name: "Chwast",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "ESGgfEgg",
-      name: "Chwast",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "ESGEsfdgg",
-      name: "Chwast",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-    {
-      id: "ESGEafggg",
-      name: "Chwast",
-      city: "Żarów",
-      price: 32,
-      picture: <CardImage />,
-    },
-  ];
+  });
+
+  const offers: Offer[] = response;
+
   return (
     <YourOfferGridContainer smallScreenColumns={2}>
-      {cards.map((card) => (
-        <OfferCard width="auto" color="offer" key={card.id}>
-          <ImageContainer>{card.picture}</ImageContainer>
+      {offers.map((offer) => (
+        <OfferCard width="auto" color="offer" key={offer._id}>
+          {/* <ImageContainer>{offer.photos[0].url}</ImageContainer> */}
+          <ImageCard
+            alt="plantPhoto"
+            src={offer.photos[0].url}
+            height="auto"
+          ></ImageCard>
           <FlexWrapper justifyContent="space-between">
             <div>
               <Typography fontSize="xxs" fontSizeMobile="xs">
-                {card.name}
+                {offer.title}
               </Typography>
               <Typography fontSize="xxxs" fontSizeMobile="xs">
-                {card.city}
+                {offer.city}
               </Typography>
             </div>
             <div>
               <PriceTypography fontSize="xxs" fontSizeMobile="xs">
-                {card.price}$
+                {offer.price}$
               </PriceTypography>
               <Icons />
```

Comment:
I am only wondering why we are showing these icons in offers list? 🤔

Reply:
I deleted them in branch I doing right now